### PR TITLE
Update test documentation and fix tests on Python 3.10

### PR DESCRIPTION
**Pull request by dnicolson**
This pull request updates the test documentation after changes in https://github.com/lra/mackup/commit/c02a0610faa222c606b96532e9f9d214fafd2b78. Some dependencies were changed to make the tests run on Python 3.10.

### CODE REVIEW

File: .github/workflows/install.yaml [hunk 1]
```diff
@@ -10,9 +10,9 @@ jobs:
       matrix:
         python-version:
           - 2.7
-          - 3.6
```
Comment (lra) [inline .github/workflows/install.yaml:13]: Why removing 3.6 and 3.7?

Comment (dnicolson) [reply .github/workflows/install.yaml:13]: It's somewhat of an arbitrary choice because 3.9 and 3.10 were added. Feel free to leave them in.

Comment (lra) [reply .github/workflows/install.yaml:13]: I can't update your branch, I'd have to create my own PR, your call.

**Comment by lra** [comment]
Looks like it requires more work, see https://github.com/lra/mackup/pull/1837

**Review by lra — APPROVED** [review]
Thanks!

**Pull request merged**